system_test/only_local/aest_heavy_sync_SUITE.erl

```diff
@@ -138,7 +148,7 @@ many_spend_txs(Cfg) ->
     ct:log("MicroBlock distribution ~p", [FoundNode3]),
 
     %% It's to be expected that at least one micro block contains max nr of transactions
-    ?assertEqual(lists:max(maps:values(FoundNode3)), TxsPerMB),
+    ?assertEqual(TxsPerMB, lists:max(maps:values(FoundNode3))),
```

Comment:
Wasn't this deemed very brittle in the internal discussion ?!

Reply:
Well, I picked the nonce carefully, but right so, I changed it to be max 2 transactions of.